xml/System.Text/Rune.xml

```diff
@@ -1694,10 +1694,20 @@ For more information, see <xref:System.Char.IsWhiteSpace%2A?displayProperty=name
         <Parameter Name="obj" Type="System.Object" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="other">The object to compare with the current instance.</param>
+        <summary>Compares the current instance to the specified object.</summary>
```

Comment:
I wonder if `/// <inheritdoc>` works here

Reply:
It probably does work, but I found a bug in the tool that made it crash when encountering a special character. So I transcribed it manually.